.github/workflows/theseus-build.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - main
+      - prod
```

Comment:
As far as I can tell, `prod` is used as a sort of master Git Flow branch, meaning that it always trails behind `main`, the develop branch in Git Flow terminology. So it doesn't seem necessary to rebuild the application on pushes to `prod`, because actual releases are covered by tag pushes, and all features merged to `main` are already covered there.